pkg/gofr/datasource/file/ftp/fs.go

```diff
@@ -79,7 +92,12 @@ func (f *fileSystem) Connect() {
 
 	ftpServer := fmt.Sprintf("%v:%v", f.config.Host, f.config.Port)
 
-	defer f.processLog(&FileLog{Operation: "Connect", Location: ftpServer, Status: &status, Message: &msg}, time.Now())
+	defer f.processLog(&FileLog{
+		Operation: "Connect",
+		Location:  ftpServer,
+		Status:    &status,
+		Message:   &msg,
```

Comment:
`msg` is never overloaded

Why returning an empty string, I mean the field could stay unset no?